src/parquet/util/rle-encoding.h

```diff
@@ -26,6 +26,7 @@
 #include "parquet/util/compiler-util.h"
```

Comment:
Note, I doubt that improving bpacking.h is the quickest/most efficient course of action. @lemire has done more work on bit-packing code, and a SIMD version can be found at https://github.com/lemire/simdcomp